packages/compass/src/app/components/workspace.tsx

```diff
@@ -86,7 +92,7 @@ export default function Workspace({
           ]}
         >
           <WorkspacesPlugin
-            initialWorkspaceTabs={[{ type: 'My Queries' }]}
+            initialWorkspaceTabs={[{ type: 'Welcome' }]}
```

Comment:
Shouldn't this be behind a feature toggle?

Reply:
Yea it should be but you shouldn't see this anymore. I was more like experimenting with this.